tests/integration/components/project-settings/integrations/github-project-test.js

```
@@ -0,0 +1,569 @@
+        )
+        .hasText(t('githubNoProject'));
+
+      assert
+        .dom(
+          '[data-test-projectSettings-integrations-githubProject-orgSettingsLink]'
+        )
+
+        .hasText(t('clickingHere'))
+        .hasAttribute(
+          'href',
+          new RegExp('organization/settings/integrations', 'i')
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)